webrender/src/prim_store.rs

```diff
@@ -138,6 +138,7 @@ pub struct PrimitiveMetadata {
     pub cpu_prim_index: SpecificPrimitiveIndex,
     pub gpu_location: GpuCacheHandle,
     pub clip_task_id: Option<RenderTaskId>,
+    pub is_clear: bool,
```

Comment:
This should be in the ```RectanglePrimitive``` type since it's not common to all primitives.

Reply:
How do I get access to the `RectanglePrimitive` inside `get_blend_mode`?

Reply:
Something like:

```let rect = &self.cpu_rectangles[metadata.cpu_prim_index.0];```

should work. The ```TextRun``` primitive in the same function does something similar.

Reply:
Thanks!